jenkins/pipelines/cd/tiup/tiup-mirror-update-multi-products.groovy

```diff
@@ -549,7 +534,7 @@ node("build_go1130") {
                         [$class: 'BooleanParameterValue', name: 'ARCH_MAC', value: params.ARCH_MAC],
                         [$class: 'BooleanParameterValue', name: 'ARCH_MAC_ARM', value: params.ARCH_MAC_ARM],
```

Comment:
Can we move it into the stage "TiUP build prometheus".

Reply:
done